src/manager/agents/init-module/module/agents/sc_component_manager_agent_init.cpp

```diff
@@ -0,0 +1,37 @@
+#include <sc-agents-common/utils/AgentUtils.hpp>
+#include "utils/sc_component_manager_command_init.hpp"
+
+#include "sc_component_manager_agent_init.hpp"
+
+using namespace initModule;
+using namespace keynodes;
+
+SC_AGENT_IMPLEMENTATION(ScComponentManagerInitAgent)
+{
+  keynodes::ScComponentManagerKeynodes::InitGlobal();
+  ScAddr const & actionAddr = otherAddr;
+
+  if (!CheckAction(actionAddr))
+  {
+    return SC_RESULT_OK;
+  }
+
+  SC_LOG_DEBUG("ScComponentManagerInitAgent started");
+
+  //  TODO: change config_path
+  std::string const config_path = "/home/anna/projects/ostis-metasystem/sc-models/knowledge-base";
```

Comment:
```suggestion
  std::string const config_path = ScMemory::ms_configPath;
```
Then create a config object and from this object get the kb field from the sc-component-manager section. There seems to be such a logic in Main